### PR TITLE
feat: config tags. remove namespace & account from config items. rename tags->labels

### PR DESCRIPTION
**Pull request by adityathebe**
merge after: https://github.com/flanksource/duty/pull/651

resolves: https://github.com/flanksource/config-db/issues/463

### CODE REVIEW

File: api/v1/interface.go [hunk 1]
```diff
@@ -286,7 +377,7 @@ func NewScrapeResult(base BaseScraper) *ScrapeResult {
 	return &ScrapeResult{
 		BaseScraper: base,
 		Format:      base.Format,
-		Tags:        base.Tags,
+		Labels:      base.Labels,
```
Comment (moshloop) [inline api/v1/interface.go:380]: We need to be able to scrape both tags and labels

File: fixtures/aws.yaml [hunk 2]
```diff
@@ -43,8 +43,8 @@ spec:
           - filter: config_type == 'AWS::EKS::Cluster'
             expr: |
               [
-                {"type": "Kubernetes::Cluster","labels": {"aws/account-id": tags['account'],"eks-cluster-name": tags["alpha.eksctl.io/cluster-name"]}},
-                {"type": "Kubernetes::Node","labels": {"aws/account-id": tags['account'],"alpha.eksctl.io/cluster-name": tags["alpha.eksctl.io/cluster-name"]}}
+                {"type": "Kubernetes::Cluster","labels": {"aws/account-id": labels['account'],"eks-cluster-name": labels["alpha.eksctl.io/cluster-name"]}},
+                {"type": "Kubernetes::Node","labels": {"aws/account-id": labels['account'],"alpha.eksctl.io/cluster-name": labels["alpha.eksctl.io/cluster-name"]}}
```
Comment (moshloop) [inline fixtures/aws.yaml:47]:
```suggestion
                {"type": "Kubernetes::Cluster","tags": {"account": labels['account'],"cluster": labels["alpha.eksctl.io/cluster-name"]}},
                {"type": "Kubernetes::Node","tags": {"account": labels['account'],"cluster": labels["alpha.eksctl.io/cluster-name"]}}
```

File: scrapers/kubernetes/kubernetes.go [hunk 3]
```diff
@@ -174,23 +174,23 @@ func extractResults(ctx context.Context, config v1.Kubernetes, objs []*unstructu
 
 		var (
 			relationships v1.RelationshipResults
-			tags          = make(map[string]string)
+			labels        = make(map[string]string)
 		)
 
 		if obj.GetNamespace() != "" {
-			tags["namespace"] = obj.GetNamespace()
+			labels["namespace"] = obj.GetNamespace()
```
Comment (moshloop) [inline scrapers/kubernetes/kubernetes.go:181]: namespace is a tag

File: scrapers/kubernetes/kubernetes.go [hunk 4]
```diff
@@ -247,7 +247,7 @@ func extractResults(ctx context.Context, config v1.Kubernetes, objs []*unstructu
 								clusterScrapeResult["cluster-name"] = clusterName
 							}
 
-							cluster.Tags["eks-cluster-name"] = clusterName
+							cluster.Labels["eks-cluster-name"] = clusterName
```
Comment (moshloop) [inline scrapers/kubernetes/kubernetes.go:250]:
```suggestion
							cluster.Tags["cluster"] = clusterName
```

File: scrapers/kubernetes/kubernetes.go (hunk 3 above)
Comment (adityathebe) [reply scrapers/kubernetes/kubernetes.go:181]: It's added as a tag below. Looks like we were setting the namespace tag twice before.

File: scrapers/azure/azure.go [hunk 5]
```diff
@@ -154,7 +154,7 @@ func (azure Scraper) Scrape(ctx api.ScrapeContext) v1.ScrapeResults {
 		// Post processing of all results
 		for i := range results {
 			// Add subscription-id tag to all the resources
-			results[i].Tags = collections.MergeMap(results[i].Tags, map[string]string{
+			results[i].Labels = collections.MergeMap(results[i].Labels, map[string]string{
```
Comment (moshloop) [inline scrapers/azure/azure.go:157]: The subscription id / name and resource group should be tags

**Pull request merged**